company_website/static/how_we_work/how_we_work.sass

```diff
@@ -92,6 +92,64 @@
             display: None
 
 
+.how-we-work-playbook-header
+    .section-header
+        $description-total-height: 140px
+        $description-content-height: 100px
+        $description-text-offset: 16px
+        $description-vertical-padding: calc(#{$description-total-height} - #{$description-content-height} - #{$description-text-offset})
+        $description-background-color: $blue-color
+        $description-text-color: $orange-color
```

Comment:
Such things shouldn't be included here. It should be in the top of a file. It is better to defined such variables on the top of a file

Reply:
Done